book/intro.md

```diff
@@ -65,9 +69,8 @@ both intentional and accidental. And everywhere you look, you see the evolution
 and history of the web wrapped up in one codebase. But most of all, it's
 fun and endlessly interesting.
 
-I've also learned to appreciate the elegance and importance of these systems we
-have built, and their place in history and the economy. Let's go into those
-now.
+So that's how I fell in love with web browsers---and I hope that after reading
```

Comment:
```suggestion
So that's how I fell in love with web browsers---now let's start your journey towards loving them too. :)
```

Reply:
This suggestion helps to better to connect to "we're not done, let's continue the chapter!"